html/components/masthead.stories.js

```diff
@@ -1003,10 +999,10 @@ export const extended = () => {
               </a>
             </div>
 
-            <ul class="sprk-c-Dropdown__links">
-              <li class="sprk-c-Dropdown__item">
+            <ul class="sprk-c-Masthead__selector-dropdown-links">
+              <li class="sprk-c-Masthead__selector-dropdown-item">
                 <a
-                  class="sprk-c-Dropdown__link sprk-u-ptm"
+                  class="sprk-c-Masthead__selector-dropdown-link sprk-u-ptm"
```

Comment:
Since we have our own class now, we will want to remove this padding utility that applies `!important` CSS and add padding to our own class. You'll want to try to follow the CSS best practice with single directional spacing. So this setting was a padding top, likely due to dropdown limitations that we no longer have. So you would update the styling here to use padding in a different way if can.

Reply:
Resolved by Jane's approval to make things a consistent 16px vertical padding.

Reply:
Hey @RVMendoza , I am still seeing the !important CSS utility here, `sprk-u-ptm`. Can you remove that and add it to the selector-dropdown-link or elaborate more on what you mean by resolved by the 16px vertical padding.